scripts/certgen/gen_all_certs.ps1

```diff
@@ -6,12 +6,12 @@ $DAYS_TLS=365
 $DAYS_UPLOAD=365
 
 # configure the DN
-$env:OSSL_COUNTRY_NAME="XA"
-$env:OSSL_STATE_NAME="Test State"
-$env:OSSL_LOCALITY_NAME="Geneva"
-$env:OSSL_ORGANIZATION_NAME="WHO"
-$env:OSSL_ORGANIZATIONAL_UNIT_NAME="R&D"
-# $env:OSSL_COMMON_NAME="WHO International" # default entry of the corresponding config file will be used
+$env:OSSL_COUNTRY_NAME
```

Comment:
How are these environment variables configured in Windows? DN_template.cnf does not apply and there does not seem to be another DN Config

Reply:
We have instructed on **concepts_onboarding_checklist.md** documentation  to **use current PowerShell session as below **

You can modify the environment variables directly within the script or set them in your current PowerShell session:

1.Open PowerShell on your Windows machine.

2.Set the required environment variables.

```
$env:OSSL_COUNTRY_NAME="XC"
$env:OSSL_STATE_NAME="Test State"
$env:OSSL_LOCALITY_NAME="TEST"
$env:OSSL_ORGANIZATION_NAME="WHO"
$env:OSSL_ORGANIZATIONAL_UNIT_NAME="RND"
$env:OSSL_COMMON_NAME="NationXC_TNP"
```